Implement close/open sidebar

TyGriffin115 opened the pull request:
Took the Persistent Drawer Component from Material-UI and integrated it into our existing code to allow for the sidebar to be open or closed. Also added an AppBar to the top of the scenarioEditor, which is the red bar seen in the GIF.  Fixed two compiler warnings. 

![PersistentDrawer2](https://user-images.githubusercontent.com/54215233/98483117-e6dd2200-21d3-11eb-841d-0f2d467b7b83.gif)

TyGriffin115 committed:
Implemented Persistent Drawer for the Dynamic sidebar

Took the Persistent Drawer Component from Material-UI and integrated it into our existing code to allow for the sidebar to be open or closed. Also added an AppBar to the top of the scenarioeditor.

TyGriffin115 committed:
Fixed cmd warnings

TyGriffin115 committed:
Updated the AppBar with 3 new features

Changed the text color of "Hide Menu" to be darker.
Made the title of "Ethisim Scenario Editor" larger on the appBar.
Added a new Return to Dashboard button that redirects user to the dashboard when clicked.

TyGriffin115 commented:
This is a visual of what I added in the third commit. 
![PersistentDrawer3](https://user-images.githubusercontent.com/54215233/98489580-9c23d000-21fc-11eb-837a-3ba9906e1401.gif)

eneadodi commented:
Real good job Tyler! If you have some extra time, can you also include another icon button (icon being the default icon picture we have for stakeholders) in the top nav bar that simply returns you to the home page? We can add extra functionality to that later such as a dropdown menu that allows for log out and settings.

TyGriffin115 committed:
Updated the UI for the dashboard

Added a navbar for the dashboard
Added a stakeholder Icon Button with a dropDown menu
Logout button on dropdown menu redirects user to the home page

TyGriffin115 commented:
Here is the updated UI for the dashboard
![NavBarDashboard2](https://user-images.githubusercontent.com/54215233/98770079-66731880-23af-11eb-84e3-e198dc0e419c.gif)

Enoch-Hsiao reviewed (approved):
Maybe we can make the profile picture more evident that it's a button that can be clicked? Either than that looks really clean.
